src/components/auction/OrderbookWidget/index.tsx

```diff
@@ -140,8 +140,8 @@ const processData = (
         // Data for representation
         priceNumber: price,
         totalVolumeNumber: totalVolume,
-        priceFormatted: price.toString(),
-        totalVolumeFormatted: totalVolume.toString(),
+        priceFormatted: price.toFixed(2),
```

Comment:
Yes, this is a nice fix. I think I would prefer toSignificant(6), though.

Reply:
Take into account the variable is a number, not a Fraction object